packages/app/src/subpages/notebookSelector/pages/AddNotebook.tsx

```
@@ -0,0 +1,226 @@
+  return (
+    <>
+      <button
+        className="flex items-center opacity-50 hover:opacity-80 duration-200 cursor-pointer text-sm"
+        onClick={() => {
+          goBack();
+        }}
+      >
+        <ArrowLeftIcon className="inline h-4 w-4 mr-1" />
+        戻る
+      </button>
+      <h2 className="text-xl opacityy-90 mt-1">新しいノートブック</h2>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)